config/middlewares.js

```diff
@@ -22,7 +22,44 @@ const DevLaunchersSubdomains = [
 
 module.exports = [
   'strapi::errors',
-  'strapi::security',
+
+  {
+    name: "strapi::security",
+    config: {
+      contentSecurityPolicy: {
+        useDefaults: true,
+        directives: {
+          "connect-src": ["'self'", "https:"],
```

Comment:
Where did you get these values from? Is there any documentation that you can link to?